src/databricks/labs/ucx/account.py

```diff
@@ -258,3 +259,67 @@ def manual_workspace_info(self, prompts: Prompts):
         for installation in Installation.existing(self._ws, 'ucx'):
             installation.save(workspaces, filename=AccountWorkspaces.SYNC_FILE_NAME)
         logger.info("Synchronised workspace id mapping for installations on current workspace")
+
+
+class AccountMetastores:
+    def __init__(
+        self,
+        account_client: AccountClient,
+    ):
+        self._ac = account_client
+
+    def show_all_metastores(self, workspace_id: str | None = None):
+        location = None
+        if workspace_id:
+            logger.info(f"Workspace ID: {workspace_id}")
+            location = self._get_region(int(workspace_id))
+        logger.info("Matching metastores are:")
+        for metastore in self._get_all_metastores(location).keys():
+            logger.info(metastore)
+
+    def assign_metastore(
+        self,
+        prompts: Prompts,
+        str_workspace_id: str,
```

Comment:
```suggestion
        str_workspace_id: str | None,
```
can we add `prompts.choice(...)` to select a workspace if it's not specified?